pkg/tsdb/influxdb/influxql/converter/testdata/influx_select_value_from_*-frame.jsonc

```diff
@@ -0,0 +1,128 @@
+//  🌟 This was machine generated.  Do not edit. 🌟
+//  
+//  Frame[0] 
+//  Name: cpu.mean
+//  Dimensions: 2 Fields by 4 Rows
+//  +-------------------------------+-------------------+
+//  | Name: Time                    | Name: Value       |
```

Comment:
when looking at the input vs output, i would expect the parser to produce a frame with name "cpu" and fields matching the names in:
```
          "name": "cpu",
          "columns": [
            "time",
            "mean_usage_guest",
            "mean_usage_nice",
            "mean_usage_idle"
          ],
```
if we then need to split that in to [timeseries multi](https://grafana.github.io/dataplane/timeseries#time-series-multi-format-timeseriesmulti), that is a straightforward task (but should not be done as part of parsing)